docs/integrations/databases/opentelemetry/elasticsearch-opentelemetry.md

```diff
@@ -0,0 +1,252 @@
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ElasticSearch-OpenTelemetry/ElasticSearch-Schematics.png')} alt="Schematics" />
+
+## Fields Create in Sumo Logic for Elasticsearch
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of Elasticsearch App install if not already present:
+
+- **db.cluster.name** - User configured. Enter a name to identify this Elasticsearch cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **db.system** - Has a fixed value of **elasticsearch**.
+- **sumo.datasource** - Has a fixed value of **elasticsearch**.
+- **db.node.name** - Has the value of host name of the machine which is being monitored.
+
+### Prerequisites
+
+This receiver supports Elasticsearch versions 7.9+
+
+If Elasticsearch security features are enabled, you must have either the monitor or manage cluster privilege. See the [Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/authorization.html) for more information on authorization and [Security privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html).
+
+Configure logging in Elasticsearch. Elasticsearch supports logging via local text log files. Elasticsearch logs have four levels of verbosity. To select a level, set loglevel to one of:
+
+1.  debug: a lot of information, useful for development/testing
+2.  verbose: includes information not often needed, but logs less than debug
+3.  notice (default value): moderately verbose, ideal for production environments
+4.  warning: only very important/critical messages are logged
+
+All logging settings are located in [Elasticsearch.conf](https://www.elastic.co/guide/en/elasticsearch/reference/current/logging.html). By default, Elasticsearch logs are stored in `/var/log/elasticsearch/ELK-<Clustername>.log`. The default directory for log files is listed in the Elasticsearch.conf file. Logs from the
+
+## Collecting Logs, Metrics & installing Elasticsearch app
+
+Here are the steps for Collecting Logs, metric and installing the app:
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+If you want to create a new Collector please select **Add a new Collector** option.
```

Comment:
```suggestion
To create a new Collector:
	
	1. Select the **Add a new Collector** option.
	2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.
	
This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
```